internals/src/macros.rs

```diff
@@ -139,9 +139,9 @@ macro_rules! debug_from_display {
 /// Asserts a boolean expression at compile time.
 #[macro_export]
 macro_rules! const_assert {
-    ($x:expr) => {{
+    ($x:expr) => {
         const _: [(); 0 - !$x as usize] = [];
```

Comment:
Sorry for adding stuff this late but I've just remembered that `panic` in `const` is stable in our MSRV so it would be very very very nice to have something like this

```rust
($x:expr $(; $message:expr)) => {
    if !$x {
        // We can't use formatting in const, only concating literals
        panic!(concat!("assertion ", stringify!($x), " failed" $(, $message)?))
    }
}
```

OK to do in a next PR though.

Reply:
Ha! Totally coincidentally I was going through easy issues to solve late in the day and picked up https://github.com/rust-bitcoin/rust-bitcoin/issues/2427 which is related to this, so I smashed them both out.